frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/inventory.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  getNodeStatusGroups,
+  getPVCStatusGroups,
+  getPVStatusGroups,
+} from '@console/internal/components/dashboard/inventory-card/utils';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboards-page/with-dashboard-resources';
+
+const cephStorageProvisiners = ['ceph.rook.io/block'];
+
+const getFilteredPVCsData = (pvcsData: K8sResourceKind[]): K8sResourceKind[] =>
+  pvcsData.filter((pvc: K8sResourceKind) =>
+    cephStorageProvisiners.includes(
+      _.get(pvc, 'metadata.annotations["volume.beta.kubernetes.io/storage-provisioner"]'),
+    ),
+  );
+
+const getFilteredPVsData = (pvsData: K8sResourceKind[]): K8sResourceKind[] =>
+  pvsData.filter((pv: K8sResourceKind) =>
+    cephStorageProvisiners.includes(
+      _.get(pv, 'metadata.annotations["pv.kubernetes.io/provisioned-by"]'),
+    ),
+  );
+
+const k8sResources: FirehoseResource[] = [
+  {
+    isList: true,
+    kind: PersistentVolumeModel.kind,
+    prop: 'pvs',
+  },
+  {
+    isList: true,
+    kind: NodeModel.kind,
+    prop: 'nodes',
+  },
+  {
+    isList: true,
+    kind: PersistentVolumeClaimModel.kind,
+    prop: 'pvcs',
+  },
+];
+
+const InventoryCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  React.useEffect(() => {
+    k8sResources.forEach((r) => watchK8sResource(r));
+    return () => {
+      k8sResources.forEach((r) => stopWatchK8sResource(r));
+    };
+  }, [watchK8sResource, stopWatchK8sResource]);
+
+  const nodes = _.get(resources, 'nodes');
+  const nodesLoaded = _.get(nodes, 'loaded');
+  const nodesData = _.get(nodes, 'data', []) as K8sResourceKind[];
+
+  const pvcs = _.get(resources, 'pvcs');
+  const pvcsLoaded = _.get(pvcs, 'loaded');
+  const pvcsData = _.get(pvcs, 'data', []) as K8sResourceKind[];
+  const filteredPVCsData = getFilteredPVCsData(pvcsData);
+
+  const pvs = _.get(resources, 'pvs');
+  const pvsLoaded = _.get(pvs, 'loaded');
+  const pvsData = _.get(pvs, 'data', []) as K8sResourceKind[];
+  const filteredPVsData = getFilteredPVsData(pvsData);
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Inventory</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody>
+        <InventoryItem
+          isLoading={!nodesLoaded}
+          kind={NodeModel}
+          resources={nodesData}
+          mapper={getNodeStatusGroups}
+        />
+        <InventoryItem
+          isLoading={!pvcsLoaded}
+          kind={PersistentVolumeClaimModel}
+          useAbbr
+          resources={filteredPVCsData}
+          mapper={getPVCStatusGroups}
+        />
+        <InventoryItem
+          isLoading={!pvsLoaded}
+          kind={PersistentVolumeModel}
+          useAbbr
+          resources={filteredPVsData}
+          mapper={getPVStatusGroups}
+        />
+      </DashboardCardBody>
+    </DashboardCard>
+  );
+};
+
+export default withDashboardResources(InventoryCard);
```

Comment:
:+1: on using the default export, I'd also suggest to name the file according to the default component, i.e.

```diff
-components/dashboard-page/storage-dashboard/inventory.tsx
+components/dashboard-page/storage-dashboard/InventoryCard.tsx
```

but that's fully up to you.